roxie/ccd/ccdfile.cpp

```diff
@@ -37,6 +37,10 @@
 #ifdef __linux__
 #include <sys/mman.h>
 #endif
+#if defined (__linux__) || defined (__APPLE__)
```

Comment:
Because syscall.h exists on apple... Not a big deal either way, I can remove if you think clearer